CartApp/CartItem.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace CartApp
+{
+    public class CartItem 
+    {
+        public int Quantity;
+        public readonly Product Product;
+        public Discount Discount = new Discount();
```

Comment:
Discount naming does'nt suggest much.... is it discount percentage ... or on amount